### PR TITLE
[TD-13300]<fix>: docker stop takes 10 seconds

### PR DESCRIPTION
**Pull request by sangshuduo**

### CODE REVIEW

File: packaging/docker/dockerbuild.sh [hunk 1]
```diff
@@ -89,7 +89,7 @@ cp -f ${comunityArchiveDir}/${pkgFile}  .
 echo "dirName=${dirName}"
 
 
-docker build --rm -f "Dockerfile"  --network=host -t tdengine/tdengine-${dockername}:${version} "." --build-arg pkgFile=${pkgFile} --build-arg dirName=${dirName}
+docker build --rm -f "Dockerfile"  --network=host -t tdengine/tdengine-${dockername}:${version} "." --build-arg pkgFile=${pkgFile} --build-arg dirName=${dirName} --build-arg cpuType=${cpuType}
```
Comment (zitsen) [inline packaging/docker/dockerbuild.sh:92]: LGTM, but the `cpuType` here we use currently is `x64` not `amd64`, which will cause 404 page error while downloading.

Comment (zitsen) [inline packaging/docker/dockerbuild.sh:92]: LGTM, but the cpuType here we use currently is x64 not amd64, which will cause 404 page error while downloading.

**Review by zitsen — COMMENTED** [review]
see my comment

File: packaging/docker/dockerbuild.sh (hunk 1 above)
Comment (sangshuduo) [reply packaging/docker/dockerbuild.sh:92]: not really. the cpuType will be re-specified to amd64 if use x64 in release script now. I verified tini-amd64 will be downloaded as expected.

**Review by zitsen — APPROVED** [review]
LGTM

File: packaging/docker/dockerbuild.sh (hunk 1 above)
Comment (zitsen) [reply packaging/docker/dockerbuild.sh:92]: That will be ok.

**Review by Eavanyuan — APPROVED** [review]
approve

**Pull request merged**